optimus/example_test_inputs.json

```diff
@@ -0,0 +1,19 @@
+{
+  "Optimus.fastq_inputs": [
+    [
+      "gs://broad-dsde-mint-dev-teststorage/10x/demo/fastqs/pbmc8k_S1_L007_R1_001.fastq.gz",
+      "gs://broad-dsde-mint-dev-teststorage/10x/demo/fastqs/pbmc8k_S1_L007_R2_001.fastq.gz",
+      "gs://broad-dsde-mint-dev-teststorage/10x/demo/fastqs/pbmc8k_S1_L007_I1_001.fastq.gz"
+    ],
+    [
+      "gs://broad-dsde-mint-dev-teststorage/10x/demo/fastqs/pbmc8k_S1_L007_R1_001.fastq.gz",
+      "gs://broad-dsde-mint-dev-teststorage/10x/demo/fastqs/pbmc8k_S1_L007_R2_001.fastq.gz",
+      "gs://broad-dsde-mint-dev-teststorage/10x/demo/fastqs/pbmc8k_S1_L007_I1_001.fastq.gz"
+    ]
+  ],
+  "Optimus.whitelist": "gs://broad-dsde-mint-dev-teststorage/10x/whitelist/737K-august-2016.txt",
+  "Optimus.tar_star_reference": "gs://broad-dsde-mint-dev-teststorage/demo/star.tar",
+  "Optimus.sample_name": "pbmc8k_test",
```

Comment:
This should say `Optimus.sample_id`

Reply:
There should also be an example `fastq_suffix`

Reply:
adjusted Optimus.sample_id. 
fastq_suffix has a default argument and should only be set in the wrapper, where it is explained.

Reply:
Oh I see! thanks!